src/app/orbit-list/orbit-list.component.html

```diff
@@ -6,15 +6,16 @@
         <th>Orbit Type</th>
         <th>Launch Date</th>
     </tr>
-    <tr class="data-row">
-        <td>GPS 938</td>
-        <td>Positioning</td>
-        <td>true</td>
-        <td>HIGH</td>
-        <td>2001-11-01</td>
+    <tr *ngFor="let newRow of satellites ">
```

Comment:
Tests are failing because the class was removed. So want to keep this:
`<tr class="data-row" *ngFor="let newRow of satellites">`